pkg/defaults/policies/files/deployment_has_ingress_network_policy.json

```diff
@@ -141,6 +141,42 @@
       ]
     }
   ],
+  "mitreAttackVectors": [
+    {
+      "tactic": "TA0001",
```

Comment:
How does not having ingress system policy indicate that an adversary is trying to gain access?